test/api_java.jl

```diff
@@ -1,4 +1,86 @@
 @testset "Java API" begin
+
+    function set_parameters(model::JavaCPOModel)
+        # Set parameters (DefaultInferenceLevel)
+        # Be careful when setting the parameter value to "Default", 
+        # because it allows the IloCP to control the value of the 
+        # parameter, as it is for the parameter 
+        # `AllMinDistanceInferenceLevel` 
+        # (https://www.ibm.com/docs/api/v1/content/SSSA5P_20.1.0/ilog.odms.cpo.help/refjavacpoptimizer/html/ilog/cp/IloCP.IntParam.html#AllMinDistanceInferenceLevel) 
+        # for instance. And thus, the parameter value retrieval may not
+        # be the same as the value entered.
+        int_parameters = Dict{String, Union{Int32, String}}(
```

Comment:
Is there a reason why you have a dictionary for integer and strings, then another one for doubles? Couldn't you have one dictionary for all three use cases?

Reply:
No particular reason. Change implemented [here](https://github.com/JuliaConstraints/CPLEXCP.jl/pull/10/commits/57e31025321f8fcc141b951c51cfd5c27c4676e4#diff-a96c425fa0553a595e9ce870fae44377a4d7077b474b08b3e6d7bd81162eb0e8R12).